### PR TITLE
BXMSDOC-7657-master: Add note regarding wrapped parameters

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [id='create-service-task-proc']
 
 = Creating service tasks
-A service task is a task that executes an action based on a web service call or in a Java class method. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. A Service Task should have one incoming connection and one outgoing connection.
+A service task is a task that executes an action based on a web service call or in a Java class method. Examples of service tasks include sending an email and logging a message when these tasks are performed by systems. You can define the parameters (input) and results (output) that are associated with a service task. You can also define wrapped parameters that contain all input into single objects. A Service Task should have one incoming connection and one outgoing connection.
```
Comment (emmurphy1) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc:4]: all inputs

Comment (csherrar) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc:4]: Agreed, Thanks Emily!

Comment (cristianonicolai) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc]: @csherrar would be interesting to also mention the new parameter that is needed to use this new functionality. 'Wrapped' needs to be set to _true_ .

Comment (csherrar) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc]: Thanks, @emmurphy1 @cristianonicolai

**Review by romartin — APPROVED** [review]

**Pull request merged**